### ISSUE TITLE
Playback speed support

### ISSUE THREAD

**Issue by defagos** [SRGSSR/srganalytics-apple#57]
Tracking must be updated if we want to support various playback rates:

- comScore provides a playback rate method to inform the SDK about speed changes. This method generates a rate event during playback if needed, otherwise the value is provided in standard comScore labels.
- Letterbox web provides this infromation to Webtrekk in a `media_playback_rate` [label](https://github.com/SRGSSR/srgletterbox-web/blob/v3.11.0/src/analytics/SRGAnalytics.js#L364). No additional events are sent when the rate is changed, the current value is just returned when an event is sent.

**Comment by defagos** [comment]
Available for review on `feature/playback-speed`.

**Comment by defagos** [comment]
During validation by comScore we were reported an issue when playback is restarted. The rate would be always incorrectly reset to `100` (i.e. 100% = 1)  in the `play` event following the restart. We wrote a UT reproducing the issue, then a fix was applied to ensure the playback rate is always correctly notified when the tracker is created again. The UT passed.

Having a fix we sent our app for validation one more time but got the same issue reporte again. And in fact, to my great surprise, the same code tested in UTs or triggered in the sample app to restart playback produced different results when comScore traffic is actually inspected. The code path followed in both cases is the same, re-creating a tracker at the end of playback with the correct effective playback speed immediately notified to the comScore SDK. And while playing again is achieved by ultimately calling the same `-play` method, the corresponding `play` comScore event contains the correct expected value in the UT, but always `100` in the app.

We don't have access to the comScore SDK implementation but I guess this might be related to how comScore keeps track of the playback rate internally. I therefore decided to apply a more reliable fix to avoid time gaps between tracker re-generation at playback end and the subsequent `play` event. I first thought about always notifying the playback rate to the SDK before all comScore event for theoretically greater consistency, but this was leading to unnecessary `playrt` events being emitted (e.g. when pausing playback), even if the rate notified to the SDK did not change.

Instead, it seems that notifying the playback rate to the SDK before `play` events only works well. This ensures the value is always consistent while avoiding unnecessary `playrt` events, as the value is directly inlined in the `play` event without generating additional `playrt` artifacts. I wrote a UT to check this behavior in the presence of several play / pause transitions are triggered. This should hopefully prevent us from modifying the code in a way that would generate unnecessary `playrt` artifacts in the future.
